### DISCUSSION TITLE
Failed to Install macOS Ventura Version 13.2.1

### DISCUSSION THREAD

**Discussion by ruhul011011** [DrDonk/unlocker#95] [General]
Result: Failed
CPU: Ryzen 9 5950X
VMware Player version: 16.2.4; using unlocker 4.2.2
Host OS: Windows 11 (22H2)
Guest OS: macOS Ventura Version 13.2.1

**Comment by DrDonk** [comment]
AMD is not supported only Intel hosts.

**Reply by ruhul011011** [reply]
I have already installed Monterey, BigSur with  this processor 
but Sonoma not working

**Comment by DrDonk** [comment]
It does not work with Ventura and Sonoma and there has been much work to try and make it work.

https://github.com/DrDonk/unlocker/issues/84

https://github.com/DrDonk/unlocker/wiki/macOS-Ventura-Guests

Please note I have stopped all development on the unlocker and only re-enabled the repo to ensure a security fix for VMware Tools was included.
